building/build-debs/homeworld-admin-tools/src/authority.py

```diff
@@ -47,7 +64,7 @@ def generate() -> None:
         cryptdir = os.path.join(d, "cryptdir")
         os.mkdir(cryptdir)
         for filename in os.listdir(certdir):
-            if filename.endswith(".pub") or filename.endswith(".pem"):
+            if filename.endswith(".pub") or (filename.endswith(".pem") and validate_pem_file(certdir, filename)):
```

Comment:
You're using validate_pem_file here as a boolean, which is declared not to return anything.

Did you test this code? I'm not quite sure why this would work.

Reply:
It works, although incorrectly.

For me:
```bash
$ python3 --version
Python 3.6.3
```

The following code prints 0.
```python
if True and None:
    print(1)
else:
    print(0)
```

Reply:
"Working, although incorrectly" is not a kind of "working."

(i.e. yes, you *can* use a None as a boolean. It evaluates to false. But I don't think you mean to do that.)